### DISCUSSION TITLE
Question: menu item click route to create page instead of index page

### DISCUSSION THREAD

**Discussion by topegret** [filamentphp/filament#4029] [Help] [answered]
new to Filament. quick question. can I custom model menu item on click behavior?  the default was goes to index route. what if I want to directly goes to create and disable index for one of my models?  thanks!

**Comment by danharrin** [comment]
Inside that resource you can use `getPages()` to point the `index` route to your create page instead of list.

But if you're getting rid of List, you probably dont need a resource at all. Create a custom page and use the form builder for your create form.

**Comment by topegret** [comment, marked as answer]
I see.  thanks!
